Packs/ExportIndicators/Integrations/ExportIndicators/ExportIndicators.py

```diff
@@ -293,23 +309,37 @@ def get_outbound_mimetype() -> str:
     return ctx.get(CTX_MIMETYPE_KEY, 'text/plain')
 
 
-def is_request_change(limit, offset, out_format=FORMAT_TEXT, last_update_data={}) -> bool:
+def is_request_change(limit, offset, out_format=FORMAT_TEXT, last_update_data={}, mwg_type: str = "string",
+                      drop_invalids=False, strip_port=False, category_default='bc_category',
+                      category_attribute='') -> bool:
     """ Checks for changes in the request params
 
     Args:
         limit (int): limit on how many indicators should be exported.
         offset (int): the index of the indicator from which the list should be exported.
         out_format (str): the requested output format.
         last_update_data (dict): the cached params for the last request.
+        mwg_type (str): the type indicated in the Mcafee Gateway format.
+        drop_invalids (bool): Whether to drop invalid indicators in PANOS format.
+        strip_port (bool): Whether to strip ports from url indicators in PANOS format.
+        category_default (str): the defualt category in ProxySG format.
+        category_attribute (str):the categories to print in ProxySG format.
 
     Returns:
         bool. True if limit/offset/out_format params have changed since the last request, False otherwise.
     """
     last_limit = last_update_data.get('last_limit')
     last_offset = last_update_data.get('last_offset')
     last_format = last_update_data.get('last_format')
+    last_mwg_type = last_update_data.get('mwg_type')
+    last_drop_invalids = last_update_data.get('drop_invalids')
+    last_strip_port = last_update_data.get('strip_port')
+    last_category_default = last_update_data.get('category_default')
+    last_category_attribute = last_update_data.get('category_attribute')
 
-    return out_format != last_format or limit != last_limit or offset != last_offset
+    return (out_format != last_format or limit != last_limit or offset != last_offset or last_mwg_type != mwg_type
+            or last_drop_invalids != drop_invalids or last_strip_port != strip_port
+            or last_category_default != category_default or last_category_attribute != category_attribute)
```

Comment:
lets move this to a function, it has the potential to get even longer, and it's hard to read it as it

Reply:
I'll try and make it look better